roles/delay-prediction-service/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install systemd files
+  template:
+    src: '{{ item }}'
+    dest: /etc/systemd/system/
+  with_items:
+    - delay-prediction-service.service
+  notify: Restart delay-prediction-service
+
+- todo: create PostgreSQL database
```

Comment:
An empty database is need here that the build script will put data in.

Reply:
Just Posgres or PostGis?

Reply:
PostGIS